runtests.py

```diff
@@ -294,8 +294,12 @@ def usage(status: int) -> None:
     print()
     print('Examples:')
     print('  %s unit-test  (run unit tests only)' % sys.argv[0])
-    print('  %s unit-test -a "*tuple*"' % sys.argv[0])
-    print('       (run all unit tests with "tuple" in test name)')
+    print('  %s testcheck  (run type checking unit tests only)' % sys.argv[0])
+    print('  %s "pytest unit-test" -a -k -a Tuple' % sys.argv[0])
```

Comment:
I tried this exact command and it seems to find no test:
```
$ ./runtests.py "pytest unit-test" -a -k -a Tuple
PARALLEL 4
SUMMARY  0 tasks selected
SUMMARY  all 0 tasks and 0 tests passed
*** OK ***
total runtime: 0.01365622878074646 sec
$ 
```
If I remove the quotes I just get a bunch of usage errors from myunit.